src/core/ext/xds/xds_cluster.cc

```diff
@@ -275,23 +275,24 @@ grpc_error_handle CdsResourceParse(
         const google_protobuf_Any* typed_config =
             envoy_config_cluster_v3_Cluster_CustomClusterType_typed_config(
                 custom_cluster_type);
-        const upb_strview aggregate_cluster_config_upb_strview =
+        const upb_StringView aggregate_cluster_config_upb_StringView =
```

Comment:
The type should change here, but not the variable name.

Reply:
Done.